gobblin-modules/gobblin-compliance/src/main/java/gobblin/compliance/ComplianceConfigurationKeys.java

```diff
@@ -0,0 +1,118 @@
+  public static final String EXTERNAL = "EXTERNAL";
+  public static final String TIMESTAMP = COMPLIANCE_PREFIX + ".job.timestamp";
+  public static final String HIVE_DATASET_WHITELIST = "hive.dataset.whitelist";
+  public static final String COMPLIANCE_DATASET_WHITELIST = COMPLIANCE_PREFIX + ".dataset.whitelist";
+  public static final String PARTITION_NAME = COMPLIANCE_PREFIX + ".partition.name";
+  public static final int TIME_STAMP_LENGTH = 13;
+  public static final String DBNAME_SEPARATOR = "__";
+
+  public static final String REAPER_RETENTION_DAYS = COMPLIANCE_PREFIX + ".reaper.retention.days";
+  public static final String CLEANER_BACKUP_RETENTION_VERSIONS =
+      COMPLIANCE_PREFIX + ".cleaner.backup.retention.versions";
+  public static final String CLEANER_BACKUP_RETENTION_DAYS = COMPLIANCE_PREFIX + ".cleaner.backup.retention.days";
+  public static final String CLEANER_TRASH_RETENTION_DAYS = COMPLIANCE_PREFIX + ".cleaner.trash.retention.days";
+
+  public static final String BACKUP_DB = COMPLIANCE_PREFIX + ".backup.db";
+  public static final String BACKUP_DIR = COMPLIANCE_PREFIX + ".backup.dir";
+  public static final String BACKUP_OWNER = COMPLIANCE_PREFIX + ".backup.owner";
+
+  public static final String TRASH_DB = COMPLIANCE_PREFIX + ".trash.db";
+  public static final String TRASH_DIR = COMPLIANCE_PREFIX + ".trash.dir";
+  public static final String TRASH_OWNER = COMPLIANCE_PREFIX + ".trash.owner";
+
+  public static final String HIVE_VERSIONS_WHITELIST = COMPLIANCE_PREFIX + ".hive.versions.whitelist";
+  public static final String SHOULD_DROP_EMPTY_TABLES = COMPLIANCE_PREFIX + ".drop.empty.tables";
+  public static final String DEFAULT_SHOULD_DROP_EMPTY_TABLES = "false";
+
+  public static final String COMPLIANCE_JOB_SIMULATE = COMPLIANCE_PREFIX + ".simulate";
+  public static final boolean DEFAULT_COMPLIANCE_JOB_SIMULATE = false;
+
+  public static final String GOBBLIN_COMPLIANCE_JOB_CLASS = COMPLIANCE_PREFIX + ".job.class";
+  public static final String GOBBLIN_COMPLIANCE_DATASET_FINDER_CLASS = COMPLIANCE_PREFIX + ".dataset.finder.class";
+
+  public static final String GOBBLIN_COMPLIANCE_PROXY_USER = COMPLIANCE_PREFIX + ".proxy.user";
+  public static final boolean GOBBLIN_COMPLIANCE_DEFAULT_SHOULD_PROXY = false;
+  public static final String GOBBLIN_COMPLIANCE_SHOULD_PROXY = COMPLIANCE_PREFIX + ".should.proxy";
+  public static final String GOBBLIN_COMPLIANCE_SUPER_USER = COMPLIANCE_PREFIX + ".super.user";
+
+  public static final String RETENTION_VERSION_FINDER_CLASS_KEY = COMPLIANCE_PREFIX + ".retention.version.finder.class";
+
+  public static final String RETENTION_SELECTION_POLICY_CLASS_KEY =
+      COMPLIANCE_PREFIX + ".retention.selection.policy.class";
+
+  public static final String DATASET_SELECTION_POLICY_CLASS = COMPLIANCE_PREFIX + ".dataset.selection.policy.class";
+  public static final String DEFAULT_DATASET_SELECTION_POLICY_CLASS = "gobblin.compliance.HivePartitionDatasetPolicy";
```

Comment:
COMPLIANCE_PREFIX = "gobblin.compliance" is prefix for properties. While for some cases like this it matches with the package name. So for all policy classes, I didn't add COMPLIANCE_PREFIX.